docs: improve readme (#1411)
ReVanced/revanced-manager: docs/README.md

   3. [🔄 Updating ReVanced Manager](2_3_updating.md)
   4. [⚙️ Configuring ReVanced Manager](2_4_settings.md)
3. [❔ Troubleshooting](3_troubleshooting.md)

> ## 👋 Developer version
> Are you a contributors? Consider checking out developer version of the [documentation](/developers/README.md).

Author: ```suggestion
## 👋 Developer version
Interested in contributing? Consider checking out developer version of the [documentation](/developers/README.md).
```

>    3. [🔄 Updating ReVanced Manager](2_3_updating.md)

Reviewer: Updating has been implemented. The file is out of date (It is also not showing up in the github review page for some reason!)

Author: Done:tm:!

   3. [🔄 Updating ReVanced Manager](2_3_updating.md)
   4. [⚙️ Configuring ReVanced Manager](2_4_settings.md)
3. [❔ Troubleshooting](3_troubleshooting.md)

## 👋 Developer version

> Interested in contributing? Consider checking out developer version of the [documentation](/developers/README.md).

Reviewer 2: How about `...? Consider checking out the developer [documentation](/developers/README.md)`

Author: ```suggestion
Interested in contributing? Consider checking out the developer [documentation](/developers/README.md)
```

Co-authored-by: Palm <palmpasuthorn@gmail.com>

> Interested in contributing? Consider checking out developer version of the [documentation](/developers/README.md).

Author: 👋 Hello!

Your suggestion has been approved and is now resolved in 944b57c336cebdc6ecd29bbde7a8b52d958986c9
